demos/link-prediction/directed-graphsage/cora-links-example.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
Other demos for directed GraphSAGE are just categorised under `graphsage`, e.g. https://github.com/stellargraph/stellargraph/blob/develop/demos/node-classification/graphsage/directed-graphsage-on-cora-example.ipynb

so maybe this can be moved to `demos/link-prediction/graphsage`?